pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -297,7 +299,7 @@ func (a *Actuator) isMachineOutdated(machineSpec *v1alpha1.AWSMachineProviderSpe
 	// as a *string, so do the same here.
 	if machineSpec.Subnet != nil {
 		if aws.StringValue(machineSpec.Subnet.ID) != instance.SubnetID {
-			return true
+			errs = append(errs, errors.Errorf("(machine subnet ID cannot be mutated from %q to %q)", instance.SubnetID, aws.StringValue(machineSpec.Subnet.ID)))
```

Comment:
Any reason errors are in parentheses?

Reply:
I wasn't sure of the best way to delimit for the benefit of the log output. There's probably a better way I'm unaware of...